Horizon/Horizon/Sources/Features/AssignmentDetails/View/AssignmentDetailsViewModel.swift

```diff
@@ -121,6 +132,16 @@ final class AssignmentDetailsViewModel {
                     self?.errorMessage = error.localizedDescription
                 }
             }.store(in: &subscriptions)
+
+        aiEvents.sink {  [weak self] event, controller in
+            switch event {
+            case .assist:
+                self?.appEnvironment.router.route(to: "/tutor", from: controller)
+            default:
+                break
+
+            }
+        }.store(in: &subscriptions)
```

Comment:
We could just pass the `Router` as a dependency, instead of passing the whole `AppEnvironment`.

Reply:
Yes, but I need it to show and hide tab bar